my-todo/src/App.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './App.css';
+import { TodoList } from './components/TodoList';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <TodoList/>
+      </header>
+    </div>
```

Comment:
So you have index.js and then app.js.  Inside app.js you only have 1 component which is holding everything else.  I think it might be better suited if you put what's inside TodoList inside App.js